ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthPeer.java

```diff
@@ -200,12 +200,17 @@ public void removeChainEstimatedHeightListener(final long listenerId) {
 
   public void recordRequestTimeout(final int requestCode) {
     LOG.debug("Timed out while waiting for response from peer {}", this);
-    reputation.recordRequestTimeout(requestCode).ifPresent(this::disconnect);
+    reputation.recordRequestTimeout().ifPresent(this::disconnect);
   }
 
   public void recordUselessResponse(final String requestType) {
     LOG.debug("Received useless response for {} from peer {}", requestType, this);
-    reputation.recordUselessResponse(System.currentTimeMillis()).ifPresent(this::disconnect);
+    reputation.recordUselessResponse().ifPresent(this::disconnect);
```

Comment:
it's a bit confusing to have the 2 methods recordUselessResponse and recordUselessAndDisconnect - can they be combined?

Reply:
I'll clarify the method name a bit.